config/settings.yml

```diff
@@ -966,6 +966,7 @@
   :server_log_frequency: 5.minutes
   :server_log_timings_threshold: 1.second
   :server_monitor_frequency: 60.seconds
+  :server_role_monitor_frequency: 15.seconds
```

Comment:
NOTE I made this 15 seconds because that matches the worker_monitor_frequency below which is essentially how often `sync_monitor` above checked for `my_server.active_roles_changed?`